vitrivr-engine-module-fes/build.gradle

```diff
@@ -0,0 +1,52 @@
+import org.apache.tools.ant.taskdefs.condition.Os
+
+tasks.register('downloadOpenApiSpec', Exec) {
```

Comment:
Why don't you use a proper download task, see https://plugins.gradle.org/plugin/de.undercouch.download